source/_integrations/habitica.markdown

```diff
@@ -145,3 +145,10 @@ Also an event `habitica_api_call_success` will be fired with the following data:
     "id": "NEW_TASK_UUID"}
 }
 ```
+
+This Markdown will list the tasks saved in `sensor.habitica_USER_todos.attributes.items` in a card.
+
+```markdown
+{% for key, value in states.sensor.habitica_USER_todos.attributes.items() %}{% if 'text' in value | string %} {{ loop.index }}. {{ value.text }}{% endif %}
+{% endfor %}
```

Comment:
- Let's add some new lines to make the template more readable.
- The code fenced block needs to be wrapped in `{% raw %}` and `{% endraw %}` to prevent it from conflicting with the Liquid syntax on the website.